components/forms/technician-visit/FarmRedFlag.tsx

```diff
@@ -0,0 +1,88 @@
+import { useRef } from "react"
+import { useController, useFormContext } from "react-hook-form"
+import ErrorMessage from "../ErrorMeesge"
+import Label from "../Label"
+import { DISEASES, Disease, PRESENCE_OPTIONS, TechnicianVisitSchema } from "./schema"
```

Comment:
Love the constants :)

Reply:
It cannot be localized, so I removed all in #4.